tests/findIndex.spec.js

```diff
@@ -0,0 +1,30 @@
+import {findIndex} from '../src';
+
+/* eslint-disable require-jsdoc */
+const testArray = [5, 12, 8, 130, 44];
+
+describe('findIndex', () => {
```

Comment:
I reckon it's probably worth defining a spec that doesn't find an element in the test data

Reply:
Thoughts on this?

Reply:
Shit, sorry. I did change it, but it must have been doofed on the rebase

Reply:
Oh never mind, it was changed :) I changed the test to > 100 and the data value 130 to 100